### PR TITLE
Should assert that requests and responses are returned from interceptors

### PR DESCRIPTION
**Pull request by mclark4386**
also 2 typo fixes

### CODE REVIEW

File: chopper/lib/src/base.dart [hunk 1]
```diff
@@ -126,10 +126,15 @@ class ChopperClient {
         req = await i(req);
       }
     }
+      
+    assert(
+      req == null,
```
Comment (Kavantix) [inline chopper/lib/src/base.dart]: This comparison is wrong

File: chopper/lib/src/base.dart [hunk 2]
```diff
@@ -146,10 +151,15 @@ class ChopperClient {
         res = await i(res);
       }
     }
+      
+    assert(
+      res == null,
```
Comment (Kavantix) [inline chopper/lib/src/base.dart]: This comparison is wrong

**Comment by Kavantix** [comment]
@lejard-h can you revert this pull request (see comments) this breaks all interceptors

**Comment by lejard-h** [comment]
It's already fix on master

**Pull request merged**